client/src/components/ColorList.js

```diff
@@ -78,6 +108,37 @@ const ColorList = ({ colors, updateColors }) => {
       )}
       <div className="spacer" />
```

Comment:
For some reason, when I view the website, this spacer forces the "Add Color" form below the bottom of the page.

I just commented out this `<div/>`, and the add color form displayed properly 👍